src/nginxconfig/util/angular_backwards_compatibility.js

```diff
@@ -45,6 +45,7 @@ const globalMap = {
 
     python_server:              ['python', 'pythonServer'],
 
+    disableHtmlCaching:         ['performance', 'disableHtmlCaching', oldBool],
```

Comment:
```suggestion
```
This doesn't need to be included here as this was never a feature in the Angular version of the tool